src/pydot/core.py

```diff
@@ -569,7 +569,7 @@ def set_parent_graph(self, parent_graph: Common | None) -> None:
         self.obj_dict["parent_graph"] = parent_graph
 
     def get_parent_graph(self) -> Graph | None:
-        return self.obj_dict.get("parent_graph", None)  # type: ignore
+        return self.obj_dict.get("parent_graph", None)
```

Comment:
So weird that this suddenly showed up in code I didn't even come anywhere near touching.